docs/developer-docs/latest/setup-deployment-guides/configurations/required/server.md

```diff
@@ -16,8 +20,9 @@ The `./config/server.js` file can include the following parameters:
-| `host`<br/><br/>❗️ _Mandatory_                                 | Host name                                                                                                                                                                                                                                                                                                                                                                   | string            | `localhost`                                                                                                                      |
-| `port`<br/><br/>❗️ _Mandatory_                                  | Port on which the server should be running.                                                                                                                                                                                                                                                                                                                                 | integer           | `1337`                                                                                                                           |
+| `host`<br/><br/>❗️ _Mandatory_                                 | Host name                                                                                                                                                                                                                                                                                                                                             | string            | `localhost`                                                                                                                      |
+| `port`<br/><br/>❗️ _Mandatory_                                  | Port on which the server should be running.                                                                                                                                                                                                                                                                                                          | integer           | `1337`                                                                                                    
+| `app.keys`<br/><br/>❗️ _Mandatory_                                  | Declare session keys (based on [Koa session](https://github.com/koajs/session/blob/master/Readme.md)), which is used by the `session` middleware for the Users and Permissions plugin and the Documentation plugin.                                                                                                                                                                                                            | string            | `APP_KEYS`    
```

Comment:
They are no default values for this parameter.
In details, when a project is created, the `.env` file is filled with the environment variable `APP_KEYS` with some auto-generated keys. So the user doesn't have to configure it the first time the app launched. However, if the user removes the `APP_KEYS` variable or removes `app.keys` from `config/server.js` then the session middleware will throw an error at startup, it doesn't fallback on a default value.
So I think you can put `undefined` as the default value